src/System.Private.Xml/src/System.Private.Xml.csproj

```diff
@@ -7,7 +7,7 @@
     <RootNamespace>System.Xml</RootNamespace>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <NoWarn>$(NoWarn),649</NoWarn>
-    <DefineConstants>$(DefineConstants);ASYNC;ASTORIA_LIGHT</DefineConstants>
+    <DefineConstants>$(DefineConstants);ASYNC;</DefineConstants>
```

Comment:
you could eliminat this define as well if it's always going to be defined. just in one place:

``` c#
C:\git\corefx\src\System.Private.Xml\src\System\Xml\Core\XmlTextReaderImplAsync.cs:
 1049              {
 1050                  int bufferSize;
 1051: #if !ASYNC
 1052                  bufferSize = XmlReader.DefaultBufferSize;
 1053  #else
```